nsxt/resource_nsxt_policy_security_policy_container_cluster_test.go

```diff
@@ -0,0 +1,144 @@
+// © Broadcom. All Rights Reserved.
+// The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+// SPDX-License-Identifier: MPL-2.0
+
+package nsxt
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+)
+
+func TestAccResourceNsxtPolicySecurityPolicyContainerCluster_basic(t *testing.T) {
+	testResourceName := "nsxt_policy_security_policy_container_cluster.test"
+
+	name := getAccTestResourceName()
+	updateName := getAccTestResourceName()
+	createDesc := "Acceptance tests description"
+	updateDesc := "Updated acceptance tests description"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccOnlyLocalManager(t)
+			testAccNSXVersion(t, "9.0.0")
```

Comment:
I think this test should also run on 4.*

Reply:
I couldn't create a fake cluster on 4.2, so testing is the issue here